src/Tribe/Plugins_API.php

```diff
@@ -136,27 +162,28 @@ public function filter_api_result( $result, $action, $args ) {
 	 *
 	 * @since 4.5.3
 	 *
-	 * @param array $product_data
+	 * @param array  $product_data
+	 * @param string $product_slug
 	 *
 	 * @return array
 	 */
-	public function build_product_data( $product_data ) {
+	public function build_product_data( $product_data, $product_slug ) {
 		$defaults = array(
 			'name'                     => null,
 			'slug'                     => null,
 			'version'                  => null,
 			'author'                   => '<a href="http://m.tri.be/19o3">Modern Tribe, Inc.</a>',
 			'author_profile'           => null,
 			'requires'                 => '3.9',
-			'tested'                   => '4.7.5',
-			'rating'                   => null,
+			'tested'                   => $GLOBALS['wp_version'],
```

Comment:
Our TEC readme's tested-up-to entry often matches that of our premium plugins so if we wanted it to be more dynamic than an outright hardcode, we could use that. Your point on on things falling out of date is a good one. Perhaps we should version_compare() `$GLOBALS['wp_version']` and the readme date and use whichever is higher?

Though...maybe I'm just overcomplicating something that doesn't really matter in the end. An end user just wants to know if the plugins are compatible with the version of WP that they are running.